### PR TITLE
[runs feed] incorporate includeRunsFromBackfills into counts query

### PR DESCRIPTION
**Pull request by jamiedemaria**
## Summary & Motivation
Incorporates whether to include/exclude subruns when getting the number of results for a query

## How I Tested These Changes
updates unit tests to assert on count

## Changelog

NOCHANGELOG

### CODE REVIEW

**Comment by jamiedemaria** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/25033?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#25033** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/25033?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#25027** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/25027?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#25024** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/25024?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#25023** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/25023?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#25005** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/25005?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jamiedemaria and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py [hunk 1]
```diff
@@ -665,11 +665,21 @@ def get_runs_feed_entries(
...
 
-def get_runs_feed_count(graphene_info: "ResolveInfo", filters: Optional[RunsFilter]) -> int:
-    should_fetch_backfills = _filters_apply_to_backfills(filters) if filters else True
+def get_runs_feed_count(
+    graphene_info: "ResolveInfo", filters: Optional[RunsFilter], include_runs_from_backfills: bool
+) -> int:
+    # the UI is oriented toward showing runs that are part of a backfill, but the backend
+    # is oriented toward excluding runs that are part of a backfill, so negate include_runs_in_backfills
+    # to get the value to pass to the backend
+    exclude_subruns = not include_runs_from_backfills
```
Comment (OwenKephart) [inline python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py:674]: should we just do `include_subruns` as the negation of `exclude_subruns`, just for consistency?

File: python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py [hunk 2]
```diff
@@ -869,10 +870,15 @@ def resolve_runsFeedOrError(
     def resolve_runsFeedCountOrError(
         self,
         graphene_info: ResolveInfo,
+        includeRunsFromBackfills: bool,
```
Comment (OwenKephart) [inline python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py:873]: same comment as above

**Review by OwenKephart — APPROVED** [review]
note naming comment, feel free to ignore if it's already been discussed

**Pull request merged**